python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -883,6 +931,8 @@ def startup(self):
             self.repository.startup()
             # All Ids could have changed
             self.changed_ids = {}
+            self.flush_thread = RegistryFlusher(self)
+            self.flush_thread.start()
```

Comment:
I would prefer to turn this off for tests in the future at least but I can make this change then if needed.